PROJECT

```diff
@@ -1,3 +1,3 @@
 version: "1"
 domain: k8s.io
```

Comment:
is this correct?

Reply:
Imo we should wait with renaming to the final tld to avoid having to rename too many times.

Reply:
agreed